build/wd_cc_benchmark.bzl

```diff
@@ -23,6 +23,15 @@ def wd_cc_benchmark(
         ],
         # Only run benchmarks when explicitly requested, at least until we have some more of them
         # and can define a benchmark suite.
-        tags = ["manual", "benchmark"],
+        tags = ["off-by-default", "benchmark"],
         **kwargs
     )
+
+    # generate benchmark report
+    native.genrule(
```

Comment:
Bazel runs genrule in the `exec` configuration, meaning that running this will require everything to be built in that configuration so we'd have to recompile a lot of code just for the benchmark targets. Workarounds exist, see the `run_binary_target` macro as well as `genrule_target` (this currently just lives in a V8 patch and is not as powerful as genrule proper).
Still approving this – there's no harm being done by adding this as-is, but generating benchmark outputs is inconvenient with just genrule. I can look at replacing this with a version that doesn't have the exec config problem in the coming days.

Reply:
my hope is that by passing binary in as src and not as exec it will use binaries built for target, and not for host. wdyt?

Reply:
Looks like that worked! That's great actually, less work for us.